src/test/java/de/uhd/ifi/se/decision/management/jira/persistence/decisiongroupmanager/TestUpdateGroupName.java

```diff
@@ -0,0 +1,43 @@
+package de.uhd.ifi.se.decision.management.jira.persistence.decisiongroupmanager;
+
+import de.uhd.ifi.se.decision.management.jira.TestSetUp;
+import de.uhd.ifi.se.decision.management.jira.model.DocumentationLocation;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeElement;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeStatus;
+import de.uhd.ifi.se.decision.management.jira.model.KnowledgeType;
+import de.uhd.ifi.se.decision.management.jira.model.impl.KnowledgeElementImpl;
+import de.uhd.ifi.se.decision.management.jira.persistence.DecisionGroupManager;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class TestUpdateGroupName extends TestSetUp {
+
+	private long id;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'id' could be replaced by a local variable.](https://app.codacy.com/manual/UHD/cures-condec-jira/pullRequest?prid=5229259)